packages/Webkul/Category/src/Observers/CategoryObserver.php

```diff
@@ -26,7 +27,11 @@ public function deleted($category)
     public function saved($category)
     {
         foreach($category->children as $child) {
-            $child->touch();
+            // Hacky trick to make this method unit-testable (instead of $child->touch()
```

Comment:
Probably it is better to adjust the test than the executed code.

Does aubtracting a second from the timestamp in the test assertion work ?

Reply:
Well, you're right. I better adjust the test in an ugly way than the code.